bloomrun.js

```diff
@@ -16,6 +16,7 @@ function BloomRun (opts) {
   this._isDeep = opts && opts.indexing === 'depth'
   this._buckets = []
   this._properties = new Set()
+  this._defaultResult = null
```

Comment:
In bloomrun you can add multiple (identical) patterns. This should apply also to the catchall.
The way patterns with only regexps can be implemented is to skip them in the indexing: from all intent and purposes are catchalls.